### DISCUSSION TITLE
No reference to bug

### DISCUSSION THREAD

**Discussion by getsueineko** [IanMitchell/sentrydiscord.dev#20] [Q&A]
When running the test, I see that the bot can send a link to the error. But when I connected it to my Sentry, this is not in the message. How can this be configured?

**TEST**
![image](https://user-images.githubusercontent.com/14312837/162735659-4b9e4b50-55cd-4ca1-a506-2f45c9926d08.png)

**REAL SYSTEM**
![image](https://user-images.githubusercontent.com/14312837/162735754-fbfbe160-f823-43ba-8446-e7a310e4ba88.png)

**Comment by IanMitchell** [comment]
I think this happens when Sentry sends a relative link - I'm a little unsure of the details behind it. I think @spiralw was seeing similar things in a self-hosted setup
